.github/workflows/conda-package.yml

```diff
@@ -215,8 +214,8 @@ jobs:
       - name: Run tests
         if: env.RERUN_TESTS_ON_FAILURE != 'true'
         run: |
-          python -m pytest -q -ra --disable-warnings -vv .
-        working-directory: ${{ env.tests-path }}
+          python -m pytest -ra -v --disable-warnings --pyargs ${{ env.MODULE_NAME }}.tests
```

Comment:
Do we need to enable verbose mode? It might be hard to open the log then.

Reply:
I think it is a good idea to disable these mods to reduce the log for public and internal CI